packages/lib/src/sesamy-components.d.ts

```diff
@@ -16,6 +16,8 @@ declare module "../../../dist/lib/sesamy-components.es.js" {
   export class Button extends HTMLElement {
     loding?: boolean;
     size?: "sm" | "md" | "lg";
+    part?: string;
+    "aria-label"?: string;
```

Comment:
Hmm.. if we extend HTML-Element we shouldn't need to add aria-label, right?

Reply:
I dont know why we extend HTMLElement in the first place.

If we do extend it we should probably allow for wildcard properties (in other words: allow all propeties a native dom element would allow)